src/features/calendar/components/EventCluster/TopBadge.tsx

```diff
@@ -1,38 +1,8 @@
 import { FC } from 'react';
-import { makeStyles } from '@mui/styles';
-import { Theme } from '@mui/material';
+import { Box } from '@mui/material';
 
 import theme from '../../../../theme';
```

Comment:
When we move away from `makeStyles`, we want to also remove using the `theme` as an imported object. Instead we want to use the `theme` that is available as a param when you pass a function to the `sx` prop! Like this, for example: 

```
<Box
...
sx={(theme) => ({color: theme.palette.primary.main, ....})
...
/>
```
 
So, the task is to remove the import of theme and insted use it like in the example above :)